.github/scripts/deploy.sh

```diff
@@ -19,8 +19,8 @@ gpg --version
 
 export ORG_GRADLE_PROJECT_asciiArmoredSigningKey=$(echo $GPG_PRIVATE_KEY | base64 -d)
 export ORG_GRADLE_PROJECT_signingPassword=$GPG_PASSPHRASE
-export ORG_GRADLE_PROJECT_ossrhUsername=$OSSRH_USERNAME
-export ORG_GRADLE_PROJECT_ossrhPassword=$OSSRH_PASSWORD
```

Comment:
These variables still present in the file https://github.com/heremaps/here-artifact-gradle-resolver/blob/65d0836f4f5df466b84e6e8909eb2369f242c9ac/.github/workflows/release.yml#L29